source/adapters/level_zero/image.cpp

```diff
@@ -675,8 +677,15 @@ urBindlessImagesUnsampledImageHandleDestroyExp(ur_context_handle_t hContext,
                                                ur_device_handle_t hDevice,
                                                ur_exp_image_handle_t hImage) {
   std::ignore = hContext;
-  std::ignore = hDevice;
-  std::ignore = hImage;
+
```

Comment:
Can you add the UR_ASSERT checks for the handles in this api?

ie:
UR_ASSERT(hContext && hDevice && hImage, UR_RESULT_ERROR_INVALID_NULL_HANDLE);

Then you can also remove the std::ignore = hContext

Reply:
Done.